waterloo-rocketry/cansw_processor_stm
# Otits initial implementation

changes
- create otit module
- create and run otit task
- register 2 default tests from `main`
- add compiler #define TEST_MODE which wraps every otit related code section

things to test for this pr:
- open serial terminal
- run the program
- verify it's printing messages like the screenshot below; note the first 3 msgs from `otitsInit()` about registering tests, then the continuous msgs from the otits task
![image](https://github.com/waterloo-rocketry/cansw_processor_stm/assets/71736183/6681e243-7744-4a59-9cc8-e3b039c43120)
